### ISSUE TITLE
Add checks for OpenJDK Style §3.2 - Package declaration

### ISSUE THREAD

**Issue by Anushreebasics** [checkstyle/checkstyle#19622]
Parent issue: #19604

### Description

Add Checkstyle checks for all rules under §3.2 - Package declaration of the OpenJDK Java Style Guidelines v6.

### Rules covered

From [§3.2 - Package declaration](https://checkstyle.org/styleguides/openjdk-java-style-v6/openjdk-styleguide.html#s3.2-package-declaration):

1. The package declaration should not be line wrapped, regardless of whether it exceeds the recommended maximum length of a line. - `NoLineWrap Check`

### Checks to add to `openjdk_checks.xml`

**TreeWalker-level:**

```xml
<module name="NoLineWrap">
  <property name="tokens" value="PACKAGE_DEF"/>
</module>
```

### Progress Tracker

| Check | Rule | PR |
|-------|------|----|
| `NoLineWrap` | Package declaration should not be line wrapped | |

**Comment by Anushreebasics** [comment]
@vivek-0509  @romani  please review

**Comment by vivek-0509** [comment]
@Anushreebasics Looks good to me please make the pr,
@romani Please take a look

**Comment by Anushreebasics** [comment]
@romani please review this issue

**Comment by romani** [comment]
Fix is merged
